modules/swagger-parser/src/main/java/io/swagger/parser/processors/ExternalRefProcessor.java

```diff
@@ -59,6 +60,10 @@ public String processRefToExternalDefinition(String $ref, RefFormat refFormat) {
                         subRef.set$ref(processRefToExternalDefinition(subRef.get$ref(), subRef.getRefFormat()));
                 }
             }
+        } else if (model instanceof ArrayModel && ((ArrayModel)model).getItems() instanceof RefProperty) {
+            RefProperty subRef = (RefProperty) ((ArrayModel)model).getItems();
+            if(isAnExternalRefFormat(subRef.getRefFormat()))
+                subRef.set$ref(processRefToExternalDefinition(subRef.get$ref(), subRef.getRefFormat()));
```

Comment:
I think here we need to add an "else" case too, like here: https://github.com/swagger-api/swagger-parser/blob/master/modules/swagger-parser/src/main/java/io/swagger/parser/processors/ExternalRefProcessor.java#L71
(perhaps you should rebase this branch on top of master)